### PR TITLE
merge margins together (instead of mixing paddings and margings), allow importance box to scale smaller

### PR DESCRIPTION
**Pull request by KillariDev**
- there's mixed uses of paddings and margins for errors. This PR separates them by margin only so the margins get merged together
- allow importance box to be smaller than 400px on small window

### CODE REVIEW

File: app/css/interceptor.css [hunk 1]
```diff
@@ -743,8 +743,7 @@ svg.spinner > circle {
 	padding: 10px;
 	margin: auto;
 	margin-bottom: 10px;
-	min-width: 400px;
-	max-width: fit-content;
+	max-width: min(400px, 100%);
```
Comment (MicahZoltu) [inline app/css/interceptor.css:746]: I'm not a fan of fixed pixel count max width.  Why is this necessary?  What does it look like on a very small screen?  What about a very big screen?  What happens in both cases if you just set it to 100%?

**Review by MicahZoltu — APPROVED** [review]
Approved because we are trying to close out issues/PRs, but I really wolud prefer to see the max-width removed.

File: app/css/interceptor.css (hunk 1 above)
Comment (KillariDev) [reply app/css/interceptor.css]: yeah, `fit-content` is actually what we want. It also fix the issue I had yesterday with this

Comment (KillariDev) [reply app/css/interceptor.css]: and get rid of the min-width that we used to have
